src/js/main/show.js

```diff
@@ -153,20 +153,52 @@ const app = new Vue({
   }
 });
 
+const searchBarApp = new Vue({
+  el: "#section-search",
+  data: {
+    search_type: "by-company",
+    keyword: "",
+  },
+  methods: {
+    onSubmit: function() {
+      if (this.search_type === "by-company") {
+        router.setRoute(`/search-and-group/by-company/${encodeURIComponent(this.keyword)}`);
+      } else if (this.search_type === "by-job-title") {
+        router.setRoute(`/search-and-group/by-job-title/${encodeURIComponent(this.keyword)}`);
+      } else {
+        router.setRoute("/latest");
+      }
+    },
+    setDefault: function() {
+      this.search_type = "by-company";
+      this.keyword = "";
+    },
+    setInputInfo: function(search_type, keyword) {
```

Comment:
I use this, and get 

![image](https://cloud.githubusercontent.com/assets/1908007/18884084/923e7af2-8518-11e6-8934-f4670470ddb8.png)

![image](https://cloud.githubusercontent.com/assets/1908007/18884107/a1e00fde-8518-11e6-9cad-e7ef61331230.png)